src/QZXing.pri

```diff
@@ -282,13 +282,28 @@ qzxing_qml {
     greaterThan(QT_VERSION, 4.7): lessThan(QT_VERSION, 5.0): QT += declarative
     greaterThan(QT_MAJOR_VERSION, 4): QT += quick
 
+    CONFIG+= plugin
+
     DEFINES += QZXING_QML
 
     HEADERS +=  \
         $$PWD/QZXingImageProvider.h
 
     SOURCES +=  \
         $$PWD/QZXingImageProvider.cpp
+
+    HEADERS += \
+        qxzingplugin.h
```

Comment:
I would propose three enhancements.

a) merge the same tags (HEADERS, SOURCES)
b) Use the same notation as the above definitions of headers and sources: 
c) rename qxzingplugin.h and qxzingplugin.cpp to QZXingPlugin.h and QZXingPlugin.cpp correspondingly. Even though what you did is correct, lets stick with camel case just for the sake of consistency.

```qml
 HEADERS +=  \
        $$PWD/QZXingImageProvider.h \
        $$PWD/QZXingPlugin.h

 SOURCES +=  \
        $$PWD/QZXingImageProvider.cpp \
        $$PWD/QZXingPlugin.cpp
```